datasets/common_gen/common_gen.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+import random
+
+import nlp
+
+random.seed(42)  # This is important, to ensure the same order for concept sets as the official script.
```

Comment:
You can use `random.Random()` to create a local random numbers generator and avoid setting the seed globally.

In your case instead of doing 

```python
random.shuffle(rand_order)
```

you could do 

```python
rng = random.Random(42)
for idx, row in enumerate(f):
    ...
    rng.shuffle(rand_order)
    ...
```

Reply:
Thanks for the suggestion but I'd prefer to keep it consistent with the original `formatter.py` (https://github.com/INK-USC/CommonGen/blob/master/dataset/final_data/formatter.py)